plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -91,16 +96,12 @@ class GeneralServiceFormSection extends Component {
     });
   }
   getRuntimeSelections(data = {}) {
-    let {container = {}, cmd, gpus} = data;
+    let {container = {}, gpus} = data;
     let isDisabled = {};
     let disabledTooltipContent;
-    let type = container.type || MESOS;
-    let image = findNestedPropertyInObject(container, 'docker.image');
-    // Single container with command and no image, disable 'DOCKER'
-    if (cmd && !image) {
-      isDisabled[DOCKER] = true;
-      disabledTooltipContent = 'If you want to use Docker Engine you have to enter a container image, otherwise please select Universal Container Runtime.';
-    }
+    let type = container != null && container.type != null ?
```

Comment:
I'm not sure how to accomplish this with lenses when the value of `data.container` is `null`, so I just replaced it with an `if`.